anmykh/homework/homework_8_persons.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+
+
+class Person:
+    def __init__(self, first_name, last_name, birth_date, job, working_years, salary, country, city, gender='unknown'):
+        self.first_name = first_name
+        self.last_name = last_name
+        self.birth_date = datetime.strptime(birth_date, "%d.%m.%Y")
+        self.job = job
+        self.working_years = working_years
+        self.salary = salary
+        self.country = country
+        self.city = city
+        self.gender = gender
+
+    def name(self):
+        return f"{self.first_name} {self.last_name}"
+
+    def age(self):
+        current_date = datetime.now()
```

Comment:
as per requiremets now - is.   2018-01-01
be more thorough with documentation,
but it is ok